physionet-django/project/validators.py

```diff
@@ -127,3 +128,19 @@ def validate_topic(value):
     """
     if not re.fullmatch(r'[a-zA-Z\d][\w -]*', value):
         raise ValidationError('Letters, numbers, spaces, underscores, and hyphens only. Must begin with a letter or number.')
+
+
+def validate_gcs_bucket_object_name(filename):
+    """
+    Validate a filename for upload to gcs bucket objects.
+    File names may be at most 256 characters long.
+    Only letters, numbers, dashes, underscores, dots, and / allowed,
+    or empty. No consecutive dots or fwd slashes.
+    """
+    if len(filename) > MAX_FILENAME_LENGTH_GCS_BUCKET_OBJECT:
+        raise ValidationError(
+            f'Invalid file name "{filename}". '
+            f'File names may be at most {MAX_FILENAME_LENGTH_GCS_BUCKET_OBJECT} characters long.'
+        )
+    if not re.fullmatch(r'[\w\-\./]*', filename) or '..' in filename or filename.startswith('/') or '//' in filename:
+        raise ValidationError('Invalid path')
```

Comment:
I'd go for something detailed (since this is a error message that is shown to the client):
```suggestion
        raise ValidationError('The filename cannot start with a forward slash, or contain contain consecutive forward slashes or dots.')
```

Reply:
Oh yes, Thats true my bad, Good Point. 

`Inavalid path`  is not a good message and doesnot even give any context to the users